File: api/LogicService/EO/Request.cs
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Bson;
using LogicService.Dto;

namespace LogicService.EO
{
    public class Request
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string? Id { get; set; }
        public string? Name { get; set; }
        public string? Phone { get; set; }
        public string? Type { get; set; }
        public byte Count { get; set; }
        public string? CarSize { get; set; }
        public Place? Origin { get; set; }
        public Place? Destination { get; set; }
        public DateTime Date { get; set; }
        public DateTime? DateEnd { get; set; }
        public string? Phone_org { get; set; }
        //  public OrganizationAdminDto? Admin { get; set; }
        public OrganizationDto? Organization { get; set; }
        public bool Executed { get; set; } = false;
        public string? Id_User { get; set; }
        public DateTime Executed_Time { get; set; }
        public string? Notes { get; set; }

        public DateTime? lastModified { get; set; }
    }
}

File: front/src/app/admin/models/request.ts
import { UserInfo } from "./userModel";

export class RequestAdmin {
    id?: string;
    name?: string;
    phone?: string;
    type?: string;
    count: number = 0;
    carSize?: string;
    origin?: Place;
    destination?: Place;
    date?: Date = null;
    dateEnd?: Date;
    phone_org?: string;
    executed?: boolean;
    executed_Time?: Date;
    status?: string;
    user?: UserInfo;
    notes?: string;
    lastModified?: Date;
}

export class OrganizationAdmin {
    Name: string = '';
    Phone: string = '';
    Email: string = '';
}

export class Place {
    name: string = "";
    city: string = "";
    lat: string = "";
    lng: string = "";
    constructor(name: string, city: string, _lat: string, _long: string) {
        this.name = name;
        this.city = city;
        this.lat = _lat;
        this.lng = _long;
    }
}